src/detect-flowbits.c

```diff
@@ -105,6 +142,15 @@ static int DetectFlowbitMatchIsset (Packet *p, const DetectFlowbitsData *fd)
 {
     if (p->flow == NULL)
         return 0;
+    if (fd->or_list_size > 0) {
+        int result = 0;
+        for (int i = 0; i < fd->or_list_size; i++) {
+            result |= FlowBitIsset(p->flow, fd->or_list[i]);
```

Comment:
I suppose the anding of result isn't useful anymore now. I think you could simply do:
```c
for (int i = 0; i < fd->or_list_size; i++) {
    if (FlowBitIsset(p->flow, fd->or_list[i]))
        return 1;
}
return 0;
```